packages/sources/finage-test/src/endpoint/forex.ts

```diff
@@ -4,6 +4,35 @@ import overrides from '../config/overrides.json'
 import { httpTransport } from '../transport/forex-http'
 import { ForexBaseEndpointTypes, forexPriceInputParameters } from './utils'
 import { wsTransport } from '../transport/forex-ws'
+import { AdapterRequest } from '@chainlink/external-adapter-framework/util'
+import { PriceAdapterRequest } from '../index'
+
+/* 
+excludesMap includes currency pairs {quote: [base ...]} which should not be inverted.
```

Comment:
Could we include details in this comment on why were are having to do this special behaviour? Will help in future when we come back to this EA